lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:fluffychat/pages/contacts/presentation/model/presentation_contact.dart';
+import 'package:fluffychat/pages/new_group/new_group.dart';
+import 'package:fluffychat/pages/new_group/new_group_info_controller.dart';
+import 'package:fluffychat/pages/new_group/widget/expansion_participants_list.dart';
+import 'package:fluffychat/widgets/setting_tile.dart';
+import 'package:fluffychat/widgets/twake_components/twake_fab.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/colors/linagora_ref_colors.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+
+class NewGroupChatInfo extends StatelessWidget {
+
+  final Set<PresentationContact> contactsList;
+
+  final NewGroupController newGroupController;
+
+  const NewGroupChatInfo({
+    super.key,
+    required this.contactsList,
+    required this.newGroupController,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(context),
+      body: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 8.0),
+        child: Column(
+          children: [
+            _buildChangeProfileWidget(context),
+            const SizedBox(height: 16.0,),
+            Text(L10n.of(context)!.addAPhoto,
+              style: Theme.of(context).textTheme.labelLarge?.copyWith(
+                color: Theme.of(context).colorScheme.onSurface,
+              ),
+            ),
+            Text(L10n.of(context)!.maxImageSize(5),
+              style: Theme.of(context).textTheme.bodySmall?.copyWith(
+                color: LinagoraRefColors.material().neutral[40],
+              ),),
+            const SizedBox(height: 32,),
```

Comment:
same

Reply:
IMO, you should review in IDE, in my side, i see no problem with indentation